.github/workflows/slither.yml

```diff
@@ -0,0 +1,59 @@
+name: Static-Analysis
+on:
+    push:
+        paths:
+            - 'contracts/**'
+            - '.github/workflows/slither.yml'
+            - '**.py'
```

Comment:
Are there any python files in this project?

Reply:
No, there are no python files. So, I deleted the code of this line